functions/alt.h

```diff
@@ -0,0 +1,18 @@
+#ifndef FUNCTIONS_ALTF_H
+#define FUNCTIONS_ALTF_H
+
+#include "svf.h"
+
+// Usage: AltF
+// return value: INTEGER
+// Returns current_alternative for use in ColorSelect<>, TrSelect<> or IntSelect<>
+
+class AltSVF {
+ public:
+    void run(BladeBase* blade) {}
+    int getInteger(int led) { return current_alternative; }
+};
+
```

Comment:
Need the "optimized specializations" from int.h (line 22/23) here to avoid SingleValueAdapter from creating an unnecessary variable.

Reply:
I think I did it right, it compiles.